commands/parse/parse.go

```diff
@@ -18,40 +17,55 @@ import (
 //This command can be used for printing structured inputs from unstructured configuration inputs.
 //Can be used with '--input' or '-i' flag.
 func NewParseCommand(osExit func(int)) *cobra.Command {
```

Comment:
It looks like we removed the usage of `osExit` in this function (thank you!), but it's still a parameter here

Reply:
Going to remove this.